.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
- [ ] [OPTIONAL] you can add this .vscode folder to your .gitignore file. this is a personal settings file and isn't needed inside the repository.